linera-witty-macros/src/wit_export/mod.rs

```diff
@@ -111,15 +111,15 @@ impl<'input> WitExportGenerator<'input> {
 
             Some(self.generate_for(export_target, &target_caller_type, exported_functions))
         }
-        #[cfg(not(feature = "wasmtime"))]
+        #[cfg(not(with_wasmtime))]
         {
             None
         }
     }
 
     /// Generates the code to export functions to a mock instance for testing.
     fn generate_for_mock_instance(&mut self) -> Option<TokenStream> {
```

Comment:
I usually agree, but there needs to be handling of the feature flags somewhere. Since this is a private function, it was simpler to stub it out than making the caller more complicated and checking all feature flags there.